api/tests/test_audit_log.py

```diff
@@ -0,0 +1,176 @@
+from django.test import Client
+from django.urls import reverse
+
+from api.factories import DeliveryLogFactory
+from api.models import DeliveryLog
+from audit_log.enums import Operation
+from audit_log.models import AuditLogEntry
+
+
+def test_visiting_delivery_log_admin_view_writes_read_log(admin_user):
+    """
+    Test that visiting the DeliveryLog admin list view writes a read log for each object
+    """
+    client = Client()
+    client.force_login(admin_user)
+
+    delivery_logs = DeliveryLogFactory.create_batch(3)
+
+    # Visit the admin list page of DeliveryLog (/admin/api/deliverylog/)
+    url = reverse("admin:api_deliverylog_changelist")
+    response = client.get(url)
+    assert response.status_code == 200
+
+    # Assert that an AuditLogEntry was created for each DeliveryLog object
+    qs = AuditLogEntry.objects.filter(message__audit_event__target__path=url)
```

Comment:
Because there is no index in the `message` JSONField in `AuditLogEntry` model this is a full table scan search i.e. very slow for anything but small number of rows. So, basically ok for tests that don't use many rows in AuditLogEntry, but can be problematic if querying would be needed with a larger number of rows. I'll add thoughts related to adding a GIN index to the `AuditLogEntry.message` JSONField to the model.